jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/TaskCommon.java

```diff
@@ -108,8 +108,12 @@ static void disableHttpLogging() {
           }
         });
 
-    // Disables Google HTTP client logging.
-    java.util.logging.Logger.getLogger(HttpTransport.class.getName()).setLevel(Level.OFF);
+    // Disable only when not using custom logger.
+    // https://github.com/GoogleContainerTools/jib/issues/2356
+    if (System.getProperty("java.util.logging.config.file") == null) {
+      // Disables Google HTTP client logging.
+      java.util.logging.Logger.getLogger(HttpTransport.class.getName()).setLevel(Level.OFF);
```

Comment:
I'm not sure I understand how this solves the problem? Does gradle have it's own custom logger? And if we disable this, does this show that weird log information that we turned this off for?

Reply:
Perhaps my code comment is confusing.

To explain,

- Google HTTP Client's `HttpTransport` class uses `JUL.Logger.getLogger(HttpTransport.class.getName())` to log detailed network traffic.
- Google HTTP Client's `HttpTransport` class output detailed network traffic as INFO level.
- In Gradle, increasing general verbosity to at least `--info` allows JUL to print INFO messages, as expected.
- Therefore, running `gradle --info` generates a lot of network traffic output. This is unnecessary clutter, so we universally disabled the JUL logging of `HttpTransport` by calling `setLevel(Level.OFF)` when we start a task. We still want this.
- Therefore, our FAQ instructions to capture detailed networks logs using JUL (via `-Djava.util.logging.config.file`) doesn't work.

Does this make it clear?

Reply:
ooooh I see, yeah sounds good.